frontend/components.js

```diff
@@ -740,7 +741,8 @@ const NavbarUnauth = () =>
 
 const NavbarAuth = connect((state) => ({
   pathname: state.router.pathname,
-}))(({pathname}) => {
+  balance: state.balance,
+}))(({pathname, balance}) => {
```

Comment:
Looks like incorrect indentation.

Reply:
but is correct

Reply:
My bad, the plus and minus signs confused me